### PR TITLE
Update code owners list

### PR DESCRIPTION
**Pull request by forrestxia**
# Description
Add more code owners

# GitHub Issues
n/a

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have inspected the Grafana dashboards to verify the data is displayed properly
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
n/a

# Manual inspection of the GUI
n/a

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -18,6 +18,12 @@
 # Sharmila Ramamoorthy (sharmilarama)
 # Evgeny Uglov (EvgenyUglov)
 # Bowen Jiang (bjiang27)
+# Tao He (taohe1012)
+# Peter Cao (P-Cao)
+# Yiming Bao (baoy1)
+# Yian Zong  (YianZong)
+# Forrest Xia (forrestxia)
 
 # for all files:
```
Comment (hoppea2) [inline .github/CODEOWNERS:27]: Please remove Michael Edegware @medegw01

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by hoppea2 — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Comment by forrestxia** [comment]
e2e pass

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Pull request merged**